utilities/sell-remaining-coins.py

```diff
@@ -26,7 +26,6 @@
 parsed_creds = load_config(creds_file)
 parsed_config = load_config(config_file)
 
-LOG_TRADES = parsed_config['script_options'].get('LOG_TRADES')
 LOG_FILE = parsed_config['script_options'].get('LOG_FILE')
 LOG_FILE_PATH = '../' + LOG_FILE

```

Comment:
Please dont hardcode path, use the os library to get path relative to where the script is being ran https://stackoverflow.com/questions/918154/relative-paths-in-python

Reply:
I haven't, I just removed the ability to turn off trade logging.
Maybe I'm just a little confused as to why any bot would want the functionality not to show when a currency is bought or sold - I feel this functionality may have made sense earlier on but not now 🙂